api/src/main/java/org/openmrs/module/appointmentscheduling/reporting/data/service/AppointmentDataService.java

```diff
@@ -8,19 +8,22 @@
 import org.openmrs.module.reporting.evaluation.EvaluationException;
 import org.openmrs.module.reporting.evaluation.parameter.Mapped;
 import org.springframework.transaction.annotation.Transactional;
+import org.openmrs.annotation.Authorized;
 
 public interface AppointmentDataService extends DefinitionService<AppointmentDataDefinition> {
 
     /**
      * @see DefinitionService#evaluate(org.openmrs.module.reporting.evaluation.Definition, org.openmrs.module.reporting.evaluation.EvaluationContext)
      */
     @Transactional(readOnly = true)
+    @Authorized()
```

Comment:
Doesn't require any privileges here for the authorization?

Reply:
After exchanging a few messages with Mark Goodrich, he felt no additional privileges are currently required until all the permissions are set up and configured properly. Also, there are non-confidential appointments, so non-privileged users should be able to access the appointment data service.

you can see the comments on the ticket here: https://issues.openmrs.org/projects/AM/issues/AM-156?filter=allopenissues

Thanks!

Reply:
Then, that's fine :smile: